src/assets/index.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
```

Comment:
All files that need to be places in root of `/dist` are in `assets` - we can move them anywhere, but if we will place them in main `/src` directory there will be some more logic to be written when we will want to move them to `/dist` directory.